addons/survey/static/src/scss/survey_views.scss

```diff
@@ -1,6 +1,27 @@
 .o_kanban_card_survey {
+    overflow: hidden;
```

Comment:
I think this is breaking the activity widget (or at least hiding the interesting part of it) + the action menu also.
To hide the left part of the trophy icon, you should apply css on the trophy itself.